dw-icon-button.js

```diff
@@ -252,56 +328,61 @@ export class DwIconButton extends buttonFocus(LitElement) {
    * @protected
    */
   _buttonStyle() {
-    if(!this.buttonSize) {
-      return '';
+    if (!this.buttonSize) {
```

Comment:
There are lots of formatting changes included in this PR. It makes review process difficult.

Standard Practices:
1. Whenever you find something isn't properly formatted; please submit a separate PR for formatting, OR Just commit them into `main` or `master` branch with proper message & `[ci skip]` tag. 
2. It seems you are using a VS Code Plugin, which does this auto-formatting. So, let's define some common tools to be used by Team, and document them somewhere, so our tool-set can be uniform across team.